api/src/Tests/Cases/PaymentTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/*.
+    require_module 'standard';
+.*/
+
+namespace App\Tests\TestCase\Controller;
+
+use Atlas\Query\Delete;
+use Atlas\Query\Insert;
+use App\Tests\Base\CiabTestCase;
+
+class PaymentTest extends CiabTestCase
+{
+
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+    }
+
+
+    protected function tearDown(): void
+    {
+        parent::tearDown();
+
+    }
+
+
+    public function testPayment(): void
+    {
+        global $BASEURL, $_ENV;
+
+        if (array_key_exists('STRIPE_PRIVATE_KEY', $_ENV)) {
+            $this->runRequest(
+                'POST',
```

Comment:
Can these arguments be put in an Array and then splatted into arguments, so they don't have to be copypasta'd?